orchestration/scripts/create-bq-dataset.sh

```diff
@@ -0,0 +1,13 @@
+set -euo pipefail
+
+# Create a dataset, default expiration of 7 days
+declare -ra BQ_CREATE=(
+  bq
+  --location=US
+  mk
+  --dataset
+  --description=${DESCRIPTION}
```

Comment:
```suggestion
  "--description=${DESCRIPTION}"
```
To handle whitespace